src/main/java/com/notably/logic/correction/distance/LevenshteinDistanceCalculator.java

```diff
@@ -7,6 +7,17 @@
  * Inspired by https://web.stanford.edu/class/cs124/lec/med.pdf.
  */
 public class LevenshteinDistanceCalculator implements EditDistanceCalculator {
+    private final boolean caseSensitive;
+
+    /**
+     * Creates a {@link LevenshteinDistanceCalculator} instance
+     *
+     * @param caseSensitive Whether or not edit distance should account for case difference
+     */
+    public LevenshteinDistanceCalculator(boolean caseSensitive) {
```

Comment:
Might wanna rename the boolean to `isCaseSensitive`

Reply:
ok!